### PR TITLE
Add New Plugin: Limit Custom Keyword Length (Min & Max Characters)

### PR DESCRIPTION
**Pull request by suryatanjung**
Limit Custom Keyword Length (Min & Max Characters) - This plugin limits the minimum and maximum number of characters for custom keyword.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,6 +239,7 @@ Jump to [0-9](#0-9) | [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f)
 - [LDAP](https://github.com/k3a/yourls-ldap-plugin) - Enables use of LDAP for user authentication.
 - [Link Anonymizer](https://github.com/katzwebservices/YOURLS-Link-Anonymizer) - Generate a link that will take you to an anonymizer service.
 - [Link List](https://gitlab.com/ruthtillman/yourls-linklist) - List recent links added, in the admin interface or on a public page.
+- [Limit Custom Keyword Length (Min & Max Characters)](https://github.com/suryatanjung/yourls-limit-custom-keyword-length) - This plugin limits the minimum and maximum number of characters for custom keyword.
```
Comment (suryatanjung) [inline README.md:242]: Add New Plugin: Limit Custom Keyword Length (Min & Max Characters)

**Review by suryatanjung — COMMENTED** [review]
Add New Plugin: Limit Custom Keyword Length (Min & Max Characters)

**Review by dgw — APPROVED** [review]
Future versions might support configuring the limits in the admin panel? But there's no reason not to accept this new plugin link now.

**Comment by suryatanjung** [comment]
thank you hope this useful for future versions

**Review by suryatanjung — COMMENTED** [review]
sorry i mistakenly closed this patch

**Review by dgw — APPROVED** [review]
I had already approved this. Please just _wait_ for another maintainer to take a look, thanks.

**Review by LeoColomb — APPROVED** [review]

**Pull request merged**